internal/cloudapi/v2/server.go

```diff
@@ -1,3 +1,4 @@
+//go:generate go run -mod=mod github.com/deepmap/oapi-codegen/cmd/oapi-codegen --package=v2 --generate types,server,spec -o openapi.v2.gen.go openapi.v2.yml
```

Comment:
This is already specified in `handler.go`.  Does adding it here provide anything extra?

Reply:
hmm... I missed this and for some reason, the generation of the API code didn't work for me without this... I'll try it again 🤔

Reply:
It all works now even without my addition. I'm not sure what I was doing wrong 🤷‍♂️  I'll drop this change.